.vscode/extensions.json

```diff
@@ -1,7 +1,7 @@
 {
-    "recommendations": [
-      "dbaeumer.vscode-eslint",
-      "esbenp.prettier-vscode"
-    ]
-  }
-  
\ No newline at end of file
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "arcanis.vscode-zipfs"
```

Comment:
Since yarn keeps all of its dependencies zipped up, apparently this is useful in allowing you to explore them.